src/engine/sparqlExpressions/SparqlExpressionPimpl.cpp

```diff
@@ -92,6 +92,12 @@ auto SparqlExpressionPimpl::getEstimatesForFilterExpression(
                                                  primarySortKeyVariable);
 }
 
+//_____________________________________________________________________________
+std::vector<PrefilterExprVariablePair>
+SparqlExpressionPimpl::getPrefilterExpressionForMetadata(bool isNegated) const {
+  return _pimpl->getPrefilterExpressionForMetadata(isNegated);
```

Comment:
always use the default value here for the call, the `pimpl` doesn't need the `isNegated`.